src/components/Comments/ListComment.js

```diff
@@ -15,6 +15,7 @@ const ListComment = ({
   findCommentAndUpdate,
   hideLeftConnector,
   navigation,
+  feedId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `feedId` as a prop in the component is a good change, as it seems to be used in child components. However, please ensure that all parent components that use this component are updated to pass this new prop.
```diff
+  feedId,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->